db/fixtures/miq_product_features.yml

```diff
@@ -5292,6 +5296,11 @@
           :description: Edit a Playbook
           :feature_type: admin
           :identifier: embedded_configuration_script_payload_edit
+        - :name: Refresh Playbooks
+          :description: Ansible Playbooks Refresh
+          :feature_type: control
+          :identifier: embedded_automation_manager_playbook_refresh
```

Comment:
@mzazrivec should this be embedded_configuration_script_payload_refresh to be consistent